subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractKotlinPluginAndroidSmokeTest.groovy

```diff
@@ -83,7 +84,7 @@ abstract class AbstractKotlinPluginAndroidSmokeTest extends AbstractSmokeTest im
         [kotlinPluginVersion, androidPluginVersion, parallelTasksInProject] << [
                 TestedVersions.kotlin.versions,
                 TestedVersions.androidGradle.versions,
-                ParallelTasksInProject.values(),
+                ParallelTasksInProject.values().take(1),
```

Comment:
❓ Why this?  Shouldn't you just reference the enum token you want by name here?

Reply:
Good catch. It's a local only edit that shouldn't have made it in the PR. Reverted!